app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -308,7 +309,9 @@ public void loadSchedule(String url_, Fetcher.Source source) throws IOException
 				loadDeox(in);
 			} else if (head.contains("begin:vcalendar")) {
 				loadIcal(in);
-			} else {
+			} else if (head.contains("{")) {
+                loadJson(in);
+            } else {
```

Comment:
The rest of the file uses tabs, makes the indents look a bit funky. [...] Oh ah, nevermind, I see that the rest of your submit uses tabs, thanks. :-)